### PR TITLE
readme: added first version of readme file

### PR DESCRIPTION
**Pull request by SENya1990**

### CODE REVIEW

**Review by metallmonster — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#Antiplagiarism
+
+Antiplagiarism is a code analysis extension tool for Visual Studio that allows to analyze solutions for copypasted code from a provided reference solution. 
+The tool was designed to be used together with the Acumatica application as a reference solution but can be used with other solutions unrelated to Acumatica as an instrument to find code duplication.
+This is similar to the Visual Studio Enterprise functionality "Code Clone Detection":
+https://docs.microsoft.com/en-us/previous-versions/hh205279(v=vs.140) 
+
+For the convenient work with the found duplicated code fragments the Antiplagiarism provides a tool window located in the VS "Tools" top level menu. 
+The results are diaplayed in a grid which supports sorting and navigation to source file by double cliccking in the location column.
```
Comment (dropsonic) [inline README.md]: typos: "diAplayed" and "clicCking"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#Antiplagiarism
+
+Antiplagiarism is a code analysis extension tool for Visual Studio that allows to analyze solutions for copypasted code from a provided reference solution.
```
Comment (dropsonic) [inline README.md]: copy-pasted

**Review by dropsonic — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SENya1990) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (SENya1990) [reply README.md]: ok

**Comment by SENya1990** [comment]
@dropsonic fixed mistypes

File: README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Antiplagiarism
+
+Antiplagiarism is a code analysis tool for Visual Studio that finds in the solution the fragments of code that have been copied from another solution. 
+The tool was designed to be used along with the Acumatica ERP application. However, the tool can be used with any other solutions as an instrument to find code duplication.
+The functionality of the tool is similar to the Code Clone Detection functionality, which is available in the Visual Studio Enterprise. (For details about this functionality, see https://docs.microsoft.com/en-us/previous-versions/hh205279(v=vs.140) .)
+
+To work with the copied code fragments that the tool has found, you can open the Antiplagiarism window from the **Tools** main menu of Visual Studio. 
+You can sort the results and open the source file by double clicking the file name in the **Location** column of the table with the results.
```
Comment (SENya1990) [inline README.md]: @k-s-popova actually there are two "location" columns - "Source Location" and "Reference Location" which allow navigation to files of source and reference solutions. Should we write about it in the ReadMe?

Comment (k-s-popova) [reply README.md]: I would stay with "Source Location". Changed the file.

**Review by dropsonic — APPROVED** [review]

**Review by dropsonic — APPROVED** [review]

**Pull request merged**